man/docker-lvm-plugin.8.md

```diff
@@ -1,68 +1,83 @@
-command. 
+which can then be bind mounted into the container using `docker run`
+command.
 
 # USAGE
-Start the docker daemon before starting the docker-lvm-plugin daemon. 
+Start the docker daemon before starting the docker-lvm-plugin daemon.
 You can start docker daemon using command:
 ```bash
-systemctl start docker 
+systemctl start docker
 ```
 Once docker daemon is up and running, you can start docker-lvm-plugin daemon
 using command:
 ```bash
 systemctl start docker-lvm-plugin
-``` 
+```
 docker-lvm-plugin daemon is on-demand socket activated. Running `docker volume ls` command
 will automatically start the daemon.
 
-Since logical volumes (lv's) are based on a volume group, it is the 
-responsibility of the user (administrator) to provide a volume group name. 
-You can choose an existing volume group name by listing volume groups on 
-your system using `vgs` command OR create a new volume group using `vgcreate` 
+Since logical volumes (lv's) are based on a volume group, it is the
+responsibility of the user (administrator) to provide a volume group name.
+You can choose an existing volume group name by listing volume groups on
+your system using `vgs` command OR create a new volume group using `vgcreate`
 command. e.g.
 ```bash
-vgcreate vg1 /dev/hda 
+vgcreate vg0 /dev/hda
 ```
-where /dev/hda is your partition or whole disk on which physical volumes 
+where /dev/hda is your partition or whole disk on which physical volumes
 were created.
 
-Add this volume group name in the config file. 
+Add this volume group name in the config file.
 ```bash
 /etc/docker/docker-lvm-plugin
 ```
 The docker-lvm-plugin also supports the creation of thinly-provisioned volumes. To create a thinly-provisioned volume, a user (administrator) must first create a thin pool using the `lvcreate` command.
 ```bash
 lvcreate -L 10G -T vg1/mythinpool
 ```
-This will create a thinpool named `mythinpool` of size 10G under volume group `vg1`.
+This will create a thinpool named `mythinpool` of size 10G under volume group `vg0`.
 NOTE: thinpools are special kind of logical volumes carved out of the volume group.
-Hence in the above example, to create the thinpool `mythinpool` you must have atleast 10G of freespace in volume group `vg1`.
+Hence in the above example, to create the thinpool `mythinpool` you must have atleast 10G of freespace in volume group `vg0`.
 
 # OPTIONS
-**-debug**=*true*|*false*
-  Enable debug logging. Default is false.
-**-version**=*true*|*false*
-  Print version information and quit. Default is false.
+**--opt size**=*size*
+  Set the size of the volume.
+**--opt thinpool**=*thinpool name*
+  Create a thinly-provisioned lvm volume.
+**--opt snapshot**=*snapshot name*
+  Create a snapshot volume of an existing lvm volume.
+**--opt keyfile**=*keyfile name*
+  Create a LUKS encrypted lvm volume.
+**--opt vg**=*volume group name*
+  Create the volume in the specified volume group.
 
```

Comment:
Same as above. `OPTIONS` should not be changed. These are options to docker-lvm-plugin daemon, and not docker volume create.